### ISSUE TITLE
Deepwater Horizon oil spill and human health

### ISSUE THREAD

**Issue by aebudden** [NCEAS/oss-2017#21]
Author: **Kirsten Dorans**
Topics: Human health

## Proposed synthesis topic
“Deepwater Horizon oil spill and human health: what is known, available human health studies, and future directions”. For the synthesis topic, I propose to do an updated synthesis of what is known about how the 2010 Deepwater Horizon oil spill impacted human health. I also hope to develop a (or expand on a pre-existing) database that could serve as a resource for the development of research projects related to human health and well-being in the Gulf of Mexico on topics relevant to the Gulf Research Program. 

## Data Acquisition and Methods
We will use the following data sources: PubMed, Google Scholar, Google, and the Consortium for Resilience Gulf Communities’ resources, among other sources. 

We will primarily use the systematic literature review analytical approach. We may also use meta-analysis, though there may not be sufficient or appropriate data for a meta-analysis. We will summarize findings in a spreadsheet and hopefully work with other groups to expand on an existing database (e.g., http://www.resilientgulf.org/resources/).

As a potential extension to this project, we could also use mapping software to create a ‘story-map’ (similar to http://eli-ocean.org/gulf/map/) to summarize (a) study locations, (b) key study focuses (exposures, health outcomes), (c) major study findings, (d) study strengths and limitations, and (e) what kind of publicly available data (e.g., environmental exposures, Census data) researchers may be able to use to expand upon ongoing studies. 

## Impact / Significance
The impact/significance of this work will be building towards an updated comprehensive summary of what is known about the human health effects of the Deepwater Horizon oil spill and what pre-existing resources are available to further study this topic. The goal will be to make this information available to researchers, funders and decision-makers, to help them determine efficient allocation of future resources towards human health-focused research relevant to the Gulf Research Program and related initiatives.

After the synthesis training, I hope that this project could be further expanded to serve as a resource for human health research in topics beyond oil spill health effects. For example, other research relevant to the Gulf Research Program may examine topics such as how ecosystem restoration projects impact human or community well-being, or how ‘green’ and ‘blue’ space impact well-being in the Gulf region. We could review and integrate into a database: (1) other human health studies in the Gulf of Mexico region (such as the Bogalusa Heart Study), and (2) ecosystem restoration projects (http://eli-ocean.org/gulf/map/). This would build off of the Gulf Research Program’s prior workshop that explored “opportunities to improve the health, well-being, and resilience of communities in the Gulf of Mexico region”.

## Research plan
To assess what is known about the human health effects of the Deepwater Horizon oil spill, we will carry out a ‘systematic review of reviews’ and an updated ‘systematic review of individual research studies’. This will expand on prior reports and reviews related to the health impacts of the the Deepwater Horizon oil spill (https://www.ncbi.nlm.nih.gov/books/NBK209920/, http://www.resilientgulf.org/resources/). 

To conduct the ‘systematic review of prior reviews’, we will use the methodology paper published by Smith et al. (https://www.ncbi.nlm.nih.gov/pubmed/21291558) to guide us. We will search the databases listed above for prior reviews or summaries of this topic (search terms to be determined). We will then summarize these reviews in a spreadsheet, providing information such as (a) publication date, (b) search dates, (c) search terms used, (d) strengths and limitations of the review.

We will also conduct an updated systematic literature review of individual research studies reporting the human health impacts of the Deepwater Horizon Oil Spill, to check for any results not included in prior reviews. We will summarize individual studies, findings, and strengths and limitations in a spreadsheet. For both of these review efforts, we will define search terms, inclusion/exclusion criteria, among other criteria. 

Finally, we will use pre-existing resources (http://www.resilientgulf.org/human-impacts-research-database/) to search for ongoing, prior and planned studies of the human health impacts of the Deepwater Horizon oil spill for inclusion in our manuscript.

We will summarize findings from the literature reviews in a manuscript and database (hopefully expanding on a pre-existing database). We will present any new information gleaned from synthesizing prior reviews on this topic and also from conducting the updated review of individual research studies. We will include summaries of ongoing, prior or planned studies on the human health effects of the Deepwater Horizon oil spill, findings to-date from these studies, and suggestions or ideas for how future research may be able to build off of pre-existing work.

**Comment by askolker** [comment]
I really like the overall approach of this project for several reasons. First, it's health related and I think it would be good if we had at least one project that was related to human health. Second, the literature approach means that this project will be robust, and sometimes interpreted data. Third, the approach to data presentation is thoughtful, and potentially impactful. I think it would be excellent if this proposal was part of the course.

**Comment by blombergbn** [comment]
I agree with askolker, and also really like this proposal. I think ours complement each other, as mine (#12) is focused on looking at economic impacts to fishery-dependent households related to the oil spill in terms of fishery closures and potential changes in targeted location and how this affects income, and in turn changes in fishery pressure across the Gulf. Mine was focused on the oyster fishery, but I think expanding this to fisheries in general (and focusing on the income aspect rather than also the change in fishery pressure) could add a nice economic perspective to overall human health and well-being.

**Comment by kdorans** [comment]
Thanks so much for your thoughtful comments, @askolker and @blombergbn. @blombergbn, I also agree that there are nice complements between your proposal (#12) and this proposal. I think #12 is great and very well thought-out! I noticed that you also mentioned aspects of your proposal might fit in nicely with #29. In the Slack #oss channel, I mentioned how #12 (and #11, #17, #31) complement #29 and might fit nicely together.
